### ISSUE TITLE
Scaup: Can ATPs choose one of the trusted servers?

### ISSUE THREAD

**Issue by brodrigu** [google/ads-privacy#28]
There is a question on how we might trust one or both of the trusted MPC servers within the scaup proposals.

Could an ATP run an MPC server themselves (or contract a 3rd party to run one for them) and specify all their models should leverage their server of choice for at least one of the two MPC servers?

**Comment by pjl-google** [comment]
Yup, that’s one of the biggest questions here and it’s going to up to browsers to decide which servers they want to trust.

My current guess is that these MPC Servers would not be run by ATPs but would instead be run by other companies, in a similar way to what’s proposed for the Aggregate Reporting Helper Servers.

We didn’t have anything in the Scaup explainer about letting ATPs choose which MPC Server provider they want to work with but I feel like that should be possible.  (So long as it’s a provider that the browsers trust.)

Thanks,
Phil
